fpga/lib/ips/dma.cpp

```diff
@@ -111,8 +115,15 @@ void Dma::setupScatterGatherRingRx() {
   if (ret != XST_SUCCESS)
     throw RuntimeError("Failed to clone BD template: {}", ret);
 
+  if (cyclic) {
+    /* Enable Cyclic DMA mode */
```

Comment:
Please use C++ style comments throughout the code.

For reference: https://en.cppreference.com/w/c/comment

```suggestion
    // Enable Cyclic DMA mode
```

Reply:
done